src/styles/globals.css

```diff
@@ -85,23 +85,25 @@ body {
   overflow-x: hidden;
 }
 
-body {
+/* body {
+  padding: 0;
+  margin: 0;
   color: rgb(var(--foreground-rgb));
   background: linear-gradient(
       to bottom,
       transparent,
       rgb(var(--background-end-rgb))
     )
     rgb(var(--background-start-rgb));
-}
+} */
 
-a {
+/* a {
   color: inherit;
   text-decoration: none;
-}
+} */
 
-@media (prefers-color-scheme: dark) {
+/* @media (prefers-color-scheme: dark) {
   html {
     color-scheme: dark;
   }
-}
+} */
```

Comment:
would recommend deleting instead of commenting out (I think this is just Next.js's default styling?)

Reply:
Thank you Max! You are correct, I initially thought someone wrote those. I have just removed all the default styling CSS in global.css